packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -16,8 +51,22 @@
       min-width: 0;
       outline-offset: -0.143em;  //Adjust the outline offset for search input in Safari.
 
+      /* stylelint-disable selector-max-compound-selectors */
+      // Match focus style and make clear button appear seamless
+      &:focus + .clear {
+        border-color: var(--terra-search-field-input-focus-clear-border-color, #4cb2e9);
+      }
+    }
+
+    .input::-ms-clear,
+    .input::-ms-reveal {
+      display: none;
+    }
+
+    // Focus style when clear button is not present
```

Comment:
It looks like this class is applied when the clear button is present

Reply:
Comment changed! [6421e1b](https://github.com/cerner/terra-core/pull/2317/commits/6421e1b29105ddc7726dccabaf5e49a2238a34df)